typescript/src/main.ts

```diff
@@ -46,6 +46,23 @@ async function main(): Promise<void> {
     } else {
         print('No projects found.\n');
     }
+
+    print('');
+    const clusters = await demo.getClusters();
+    if (clusters.length !== 0) {
+        const onlineClusters = clusters.filter((c) => c.is_online);
+        if (onlineClusters.length !== 0) {
+            const webcamClusterId = onlineClusters[0].cluster_id;
+            const webcamPrinterId = onlineClusters[0].host_printer.uuid;
+            const imageUrl = await demo.getWebcamImage(webcamClusterId, webcamPrinterId);
+            print(`Received webcam image URL: ${imageUrl}`);
+        }
+    }
+
+    print('');
```

Comment:
```suggestion
```

Reply:
The output looks crappy without and extra line there.